website/pages/blog/accounts.js-1.0-rc.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Announcing Accounts.js 1.0 Release Candidate
+tags: [accounts-js, graphql-modules, graphql]
+authors: [niccolo]
+date: 2023-11-27
+description:
+  Introducing Accounts.js 1.0 Release Candidate, an end to end authentication and accounts management solution.
+image: /blog-assets/accounts.js-1.0-rc/accounts-logo-wide.png
+thumbnail: /blog-assets/accounts.js-1.0-rc/accounts-logo-squared.png
+---
+
```

Comment:
@Urigo thanks! I've added an about me section but I ended up spoiling the content of the next article a bit :)
Currently the release is scheduled for tomorrow (January 8) but I wouldn't be surprised if Martin ends up postponing the MikroORM v6 release to Tuesday. I am partially to blame for the delay because I've uncovered some bugs while writing tests for the dataloader and I proposed a non trivial design change to fix one of them which couldn't have been implemented later in a point release.

Reply:
@Urigo MikroORM v6 has just been released so this blog post is ready to release as well!